lib/src/integrationTest/kotlin/com/lemonappdev/konsist/core/declaration/kotypedeclaration/snippet/forisgenerictype/not-generic-type.kttxt

```diff
@@ -0,0 +1,3 @@
+import com.lemonappdev.konsist.testdata.SampleType
+
+class SampleClass1(val sampleProperty1: SampleType)
```

Comment:
1. This can be tested with a basic type such as `String`
2. This can be tested with a simple property (less code that a class)